### ISSUE TITLE
ImgBot does not compress additional images from newer commits

### ISSUE THREAD

**Issue by williamli** [dabutvin/Imgbot#485]
# Reproduction

1. commit images to repo
2. push to default branch
3. ImgBot start a new pull request with a commit of all images from step 1 optimised
4. do not merge, just ignore ImgBot and continue to work, commit more code changes
5. commit a few more images
6. ImgBot does not pick up and optimise additional updates

# Expected Behaviours

ImgBot should pick up additional files and process these as a new commit / update the original PR?

**Comment by dabutvin** [comment]
Hi @williamli !
Thanks for opening this and kudos on such a clear description 🎉 

I totally get what you are after hear, right now the behavior is that whenever there is an existing `imgbot` branch open in the repo, we stop and do no more more. Once the existing PR is merged, we follow up and open a new PR with the new image files.

Have you happen to seen https://github.com/dabutvin/ImgBot/issues/352 issue to add rebase support to ImgBot? Do you feel like these requests are related or play well together? I am curious to hear your thoughts.
